arcgis-ios-sdk-samples/Scenes/Create terrain surface from a local tile package/CreateTerrainSurfaceFromLocalTilePackageViewController.swift

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import UIKit
+import ArcGIS
+
+class CreateTerrainSurfaceFromLocalTilePackageViewController: UIViewController {
```

Comment:
Can you also please add comments throughout the sample? You can look at other samples for reference. For eg. https://github.com/Esri/arcgis-runtime-samples-ios/blob/master/arcgis-ios-sdk-samples/Geometry/Cut%20geometry/CutGeometryViewController.swift